code/addons/interactions/src/components/InteractionsPanel.tsx

```diff
@@ -119,20 +126,34 @@ export const InteractionsPanel: React.FC<InteractionsPanelProps> = React.memo(
             />
           ))}
         </div>
-        {caughtException && !caughtException.message?.startsWith('ignoredException') && (
+        {caughtException && !isTestAssertionError(caughtException) && (
           <CaughtException>
             <CaughtExceptionTitle>
               Caught exception in <CaughtExceptionCode>play</CaughtExceptionCode> function
             </CaughtExceptionTitle>
-            <CaughtExceptionDescription>
-              This story threw an error after it finished rendering which means your interactions
-              couldn&apos; t be run.Go to this story&apos; s play function in {fileName} to fix.
-            </CaughtExceptionDescription>
             <CaughtExceptionStack data-chromatic="ignore">
-              {caughtException.stack || `${caughtException.name}: ${caughtException.message}`}
+              {printSerializedError(caughtException)}
             </CaughtExceptionStack>
           </CaughtException>
         )}
+        {unhandledErrors && (
+          <CaughtException>
+            <CaughtExceptionTitle>Unhandled Errors</CaughtExceptionTitle>
+            <CaughtExceptionDescription>
+              Found {unhandledErrors.length} unhandled error{unhandledErrors.length > 1 ? 's' : ''}{' '}
+              while running the play function. This might cause false positive assertions. Resolve
+              unhandled errors or ignore unhandled errors with setting the
+              <CaughtExceptionCode>test.dangerouslyIgnoreUnhandledErrors</CaughtExceptionCode>{' '}
+              parameter to <CaughtExceptionCode>true</CaughtExceptionCode>.
```

Comment:
I wonder if it would be better to point people to a link in the docs instead, which could explain things better with a code snippet. WDYT @kylegach ?

Reply:
Yes, agreed. I don't believe that's currently documented anywhere, right?

The upcoming (soon!) parameters API reference would be most appropriate, then. Does the `test` namespace correspond to anything, or is it part of the Storybook core?

Reply:
Part of storybook core, but some parts are only used by the test package, for example, we have those test parameters as well:

<img width="779" alt="image" src="https://github.com/storybookjs/storybook/assets/1035299/72f4c227-baae-4249-b4a0-ce53be00fb2c">